lib/syscall_shim/include/uk/syscall_exittab.h

```diff
@@ -0,0 +1,104 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2024, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef __UK_SYSEXITTAB_H__
+#define __UK_SYSEXITTAB_H__
+
+#include <uk/assert.h>
+
+struct uk_syscall_exit_ctx {
```

Comment:
If Simon was reviewing he would had said that we should pass `struct ukarch_execenv *` directly. I don't have a strong opinion.

Reply:
My intention was to provide a structure that would allow for easy addition of other fields/flags in the future.

Reply:
Agreed, I was referring to this discussion: https://github.com/unikraft/unikraft/pull/1472#discussion_r1700353006. We can keep it like that.